src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainParameters.cs

```diff
@@ -9,6 +9,76 @@ namespace Nethermind.Specs.ChainSpecStyle;
 
 public class ChainParameters
 {
+
+    private ulong? _dencunTransitionTimestamp;
+    private ulong? _cancunTransitionTimestamp;
+
+    public ulong? DencunTransitionTimestamp
+    {
+        get
+        {
+            if (Eip4844TransitionTimestamp == Eip4788TransitionTimestamp &&
+                Eip4788TransitionTimestamp == Eip1153TransitionTimestamp &&
+                Eip1153TransitionTimestamp == Eip5656TransitionTimestamp &&
+                Eip5656TransitionTimestamp == Eip6780TransitionTimestamp)
```

Comment:
I created the helper method called "AreHardforkTimestampsEqual" in order to accomplish this one, which compares it instead of using the ifs inside each hardfork.